packages/firebase_auth/firebase_auth_web/lib/firebase_auth_web.dart

```diff
@@ -172,11 +168,9 @@ class FirebaseAuthWeb extends FirebaseAuthPlatform {
   }
 
   @override
-  Future<bool> isSignInWithEmailLink(String app, String link) {
-    // TODO(hterkelsen): Implement this once
-    // https://github.com/FirebaseExtended/firebase-dart/issues/273
-    // is resolved.
-    throw UnimplementedError('isSignInWithEmailLink');
+  Future<bool> isSignInWithEmailLink(String app, String link) async {
```

Comment:
You're not `await`ing for anything in this method, so it doesn't need to be marked as `async`, right?

_(If this **does** need to be `async`, the [platform interface](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_auth/firebase_auth_platform_interface/lib/firebase_auth_platform_interface.dart#L110-L112) may need to be updated as well!)_

Reply:
It needs to be `async` as `auth.isSignInWithEmailLink(link)` returns a `bool` on web not a `Future<bool>` as the interface expects. I can change it to use `Future.value()` if that's preferred instead.

Reply:
Yes, please, I think you should return a Future.value, instead of changing the interface here.